vendor/engines/sanger_sequencing/app/views/sanger_sequencing/admin/batches/new.html.haml

```diff
@@ -28,7 +34,11 @@
     %tbody
       - @submissions.each do |submission|
         %tr
-          %td= link_to "Add", "#", "@click.prevent" => "addSubmission(#{submission.id})", "v-if" => "hasNotBeenAdded(#{submission.id})"
+          %td
+            %div(v-if="isInPlate(#{submission.id})")
+              .sangerSequencing--colorCoded__box(v-bind:class="colorForSubmissionId(#{submission.id})")
+              = link_to "Remove", "#", "@click.prevent" => "removeSubmission(#{submission.id})"
+            = link_to "Add", "#", "@click.prevent" => "addSubmission(#{submission.id})", "v-if" => "isNotInPlate(#{submission.id})"
```

Comment:
I thought we had a shared translation for "Add" but I don't see it. We could use one.